compiler-rt/lib/ubsan/ubsan_value.h

```diff
@@ -125,6 +138,26 @@ class TypeDescriptor {
     return 1 << (TypeInfo >> 1);
   }
 
+  const char *getBitIntBitCountPointer() const {
+    CHECK(isBitIntTy());
+    CHECK(isSignedBitIntTy());
+    // Scan Name for zero and return the next address
+    const char *p = getTypeName();
+    while (*p != '\0') {
+      p++;
```

Comment:
```suggestion
      ++p;
```
I always forget about this one: https://llvm.org/docs/CodingStandards.html#prefer-preincrement

Reply:
Agree.